include/rocksdb/options.h

```diff
@@ -348,6 +349,15 @@ struct DbPath {
   DbPath(const std::string& p, uint64_t t) : path(p), target_size(t) {}
 };
 
+// RocksDB provides the checksum handoff from write path to the storaye layer.
+// User can specify the file types they want to enable the checksum handoff
+// by setting the DBOptions::checksum_handoff_file_types.
+enum class ChecksumHandoffFileType {
```

Comment:
Why not use FileType? Wouldn't this end up there anyway (but with better-but-different names)?